ui-admin/src/study/kits/KitList.tsx

```diff
@@ -42,16 +105,43 @@ export default function KitList({ studyEnvContext }: { studyEnvContext: StudyEnv
     loadKits()
   }, [studyEnvContext.study.shortcode, studyEnvContext.currentEnv.environmentName])
 
+  const kitsByStatus = _groupBy(kits, kit => {
```

Comment:
Rather than filtering here, we could let the table do the filtering for us.  That way we can give users the option to remove the filter and see all kits in a single list too if they want (or that could be another tab)

Reply:
good point on the counts, that makes sense